driver/src/main/resources/reference.conf

```diff
@@ -18,6 +18,7 @@ sparkta {
     spark.cores.max = 4
     spark.executor.memory = 1024m
     spark.app.name = SPARKTA-context
+    spark.cassandra.connection.host = "127.0.0.1"
```

Comment:
I move all in a new PR. We must implement a function in the SDK that return the spark config for this output.